packages/palette/src/elements/TextArea/TextArea.tsx

```diff
@@ -54,6 +55,8 @@ export const TextArea: React.ForwardRefExoticComponent<
 
     const [value, setValue] = useState(defaultValue)
 
+    const textAreaName = inputProps.name || "palette-text-area"
```

Comment:
I don't think we want a default name

Reply:
I'm just using it for the label for the textarea, I believe it's useful for the screen reader according to the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

Reply:
I mean inputs should have names but we should just mimic the native API here where they are optional. There are reasons for that and having a default is not helpful for screen readers in this instance.

Reply:
Hmm, maybe instead of a name we could have a default `id`?

Reply:
Neither, actually.

Reply:
(The reason you can't have default `id`s is because they need to be unique per page, and synchronizing random IDs with SSR hydration is non-trivial)

Just remove the default name and we're good.